File: poo/Java/Vogais2/Vogais2.java
import java.util.Scanner;

public class Vogais2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        char letra = scanner.next().charAt(0);
        if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u' ||
            letra == 'A' || letra == 'E' || letra == 'I' || letra == 'O' || letra == 'U') {
            System.out.println("SIM");
        }
        scanner.close();
    }
}

File: poo/Cpp/Vogais2/Vogais2.cpp
#include <iostream>

int main() {
    char letra;
    std::cin >> letra;
    if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u' ||
        letra == 'A' || letra == 'E' || letra == 'I' || letra == 'O' || letra == 'U') {
        std::cout << "SIM" << std::endl;
    }
    return 0;
}